cypress/support/fragments/finance/funds/funds.js

```diff
@@ -79,6 +81,30 @@ export default {
       .and('have.text', fundCode.concat('-', fiscalYear));
   },
 
+  checkBudgetQuantity: (quantityValue) => {
+    cy.expect(
+      Section({ id: 'pane-budget' }).find(HTML(including('Cash balance: $' + quantityValue.toFixed(2)))).exists()
+    );
+    cy.expect(
+      Section({ id: 'pane-budget' }).find(HTML(including('Available balance: $' + quantityValue.toFixed(2)))).exists()
+    );
+  },
+
+  openTransactions: () => {
+    cy.expect(Section({ id: 'information' }).find(KeyValue('Transactions')).exists());
+    // TODO: refactor via using interactors. Simple click() doesn't work, need to find a way to work with child
+    cy.xpath(viewTransactionsLinkXpath).click();
+  },
+
+  checkTransaction: (value, fundCode) => {
+    cy.expect(
+      Pane({ id: 'transaction-results-pane' }).find(HTML(including('$' + value.toFixed(2)))).exists()
```

Comment:
the same related with repeated strings and constants

Reply:
may be you can use common interactor, please check this

Reply:
Constants added. As for the interactors - I can add TODO for it and try implement in scope of refactoring for NewFund/NewLedger class. It will require some time since I didn't find and obvious way to do it using common interactors. However in this approach we're still using search for value in specific Pane.